tests/clean_tests.cmake

```diff
@@ -1,5 +1,2 @@
-FILE(GLOB OLD_FILES "${CMAKE_CURRENT_BINARY_DIR}/../tests/Head*")
-SET(TMP_FILE "Head-tmp.txt")
-LIST(APPEND OLD_FILES ${OLD_FILES} ${TMP_FILE})
-LIST(REMOVE_DUPLICATES OLD_FILES)
-EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E remove -f ${OLD_FILES})
+file(GLOB files "${CMAKE_CURRENT_BINARY_DIR}/../tests/Head*")
+file(REMOVE ${files} "Head-tmp.txt")
```

Comment:
"Head-tmp.txt" is not mentioned anywhere else. Why this?

Reply:
I've no idea it was there.

Reply:
Yup. Seems to be a remainder of something that has disppeared.... Nice catch...